javascript-modal/main.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-console */
+var openModal = document.querySelector('.button-open');
+var closeModal = document.querySelector('.pop-button');
+var background = document.querySelector('.pop-out-background');
+
+function handleOpenBackground(event) {
+  background.style.display = 'flex';
+}
+function handleClosingBackground(event) {
+  background.style.display = 'none';
+}
```

Comment:
Rather than changing the display property directly, you should add and remove classes. This is the same as inline styling, which of course we know is **_terrible_** 💀